packages/edit-site/src/components/page-templates-template-parts/index.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * This component is a thin DataViews wrapper around templates and template parts.
```

Comment:
This is me nitpicking. When I read "thin wrapper" in the PR description I was intrigued. Now that I see it's a React component (that takes props), I find it a bit too fancy 😇